DESCRIPTION

```diff
@@ -17,7 +17,12 @@ Imports:
     docopt,
     jsonlite,
     porcelain,
-    thor
+    thor,
+    dplyr,
+    tibble,
```

Comment:
This is a dependency of dplyr but it apparently needs to be specified explicitly for `make check`